README.md

```diff
@@ -77,6 +77,39 @@ can be displayed by `kcidb-schema`.
 
 To cleanup the dataset (remove the tables) use `kcidb-cleanup`.
 
+Upgrading
+---------
+
+To upgrade the dataset schema, do the following.
+
+1. Authenticate to Google Cloud with the key file (`~/.kernelci-bq.json`
+   here):
+
+        gcloud auth activate-service-account --key-file ~/.kernelci-bq.json
+
+   or login with your credentials (entered via a browser window):
+
+        gcloud auth login
+
+2. Create a new dataset (`kernelci02` in project `kernelci` here) with the new
+   schema:
+
+        bq mk --project_id=kernelci kernelci02
+        # Using new-schema kcidb
+        kcidb-init -d kernelci02
+
+3. Switch all data submitters to using new-schema kcidb and the newly-created
+   dataset.
```

Comment:
Is there a way via BQ API to make the old dataset read-only also?

Reply:
The `bq` tool has alpha support for access control modification, but you gotta be on the list of testers to use it. Otherwise you can go to the console and drop the access control manually.

Reply:
Here's a command that should return access control information for the dataset, if you're allowed to use that feature:

    bq get-iam-policy --project_id kernelci kernelci01